releasenotes/notes/42513.yaml

```
@@ -0,0 +1,7 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: istioctl
+
+releaseNotes:
+  - |
+    **Fixed** helm chart library default behaviour to generate manifests with v1.20 Kubernetes object version if using istioctl without--cluster-specific and instead use minimum Kubernetes version defined by istioctl.
```

Small typo and possible change:
```suggestion
    **Fixed** helm chart library default behavior from generating manifests with a v1.20 Kubernetes object version when using istioctl without --cluster-specific instead using the minimum Kubernetes version defined by istioctl.
```